public/js/components/SourceFooter.js

```diff
@@ -103,6 +103,10 @@ const SourceFooter = React.createClass({
   },
 
   focusSearch(shortcut, e) {
+    if (!isEnabled("search")) {
+      return true;
+    }
+
```

Comment:
If you keep it here, this function is small so I wouldn't return early. Just wrap all the code; it's easier to see the code flow.